### ISSUE TITLE
Tutorial

### ISSUE THREAD

**Issue by mrajalko** [ariacom/Seal-Report#39]
Is there any tutorial other than quick tutorial, which is good for start, but where to find answers to slightly difficult questions like 

1. how to create columns like year / month of date columns.
2. how to create columns like amount which is computed by a formula.
3. how to create drill-down report.

all these samples are available in code but how to create on your tables, right from scratch is my question.

**Comment by ariacom** [comment]
Yes, we could enhance the tutorials (contributions are welcomed)... here are the entry points for your questions:
1) Using the server manager, open your data source, select a date time column, then use the Helper property 'Create Date columns for drill' (may work only for certain database types)
2) Check https://sealreport.org/Recipes#lineFormula
3) From a column: Click on the 'Drill Children' property to define the children, if you want to create a sub-report, use the 'Create a Sub-Report' property,

**Comment by mrajalko** [comment]
Thanks for your reply.

i would try and make some tutorials and share the link.

you have made a great product.

**Comment by ariacom** [comment]
The best is to add a Tutorial Html page in the documentation (it is part of the project), if you wish I can setup the a first page (like Recipes) and you can complete with your tutorials using Github Pull requests, just let me know

**Comment by mrajalko** [comment]
there is also some issue with Jet 4.0 in Windows 10 for access database. my suggestion would be to make a default SQL Server version

**Comment by ariacom** [comment]
yes we are thinking about giving up the MS Access demo and 32bit version...

**Comment by ariacom** [comment]
I just pull a template Tutorials.cshtml in the Documentation project, if you can fill some tutorials and pull a request, it would be great... thanks to contribute.

**Comment by janadel** [comment]
Is there any tutorial on how I can create a new user.
I managed to deploy Seal Reports (web) to a managed server but it asks for login and I don't either know a default user account or how to create a new one.
On my local computer I didn't have to login.
I apologise if I am asking for something very obvious but I am very new to Seal Reports.
Thanks

**Comment by ariacom** [comment]
This depends on the Web Security implemented.
Please check https://sealreport.org/Server#lineSecurity
